OmniBLE/Bluetooth/PeripheralManager.swift

```diff
@@ -500,6 +563,19 @@ extension CBPeripheral {
 extension PeripheralManager {
     public func runSession(withName name: String , _ block: @escaping () -> Void) {
         self.log.default("Scheduling session %{public}@", name)
+
```

Comment:
Without thinking on this too much yet - I'd think the BluetoothManger would be responsible for maintaining a connection. This seems like it is masking a state problem over there by triggering a just-in-time refresh. I'm not against this change if it is useful but do want to point that the responsibilities of the different components.